src/ast/mod.rs

```diff
@@ -4225,11 +4267,10 @@ impl fmt::Display for Statement {
             Statement::Fetch {
                 name,
                 direction,
+                position,
                 into,
             } => {
-                write!(f, "FETCH {direction} ")?;
-
-                write!(f, "IN {name}")?;
+                write!(f, "FETCH {direction} {position} {name}")?;
```

Comment:
This could probably be ` write!(f, "{position} {name}")?;`, not sure what the pro/con is on that